systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -493,7 +493,7 @@ void testSendMessagesPlainAnonymous() throws InterruptedException {
         KafkaResource.kafkaEphemeral(CLUSTER_NAME, 3).done();
         KafkaTopicResource.topic(CLUSTER_NAME, topicName).done();
 
-        KafkaClientsResource.deployKafkaClients(CLUSTER_NAME + "-" + Constants.KAFKA_CLIENTS).done();
+        KafkaClientsResource.deployKafkaClients(false, CLUSTER_NAME + "-" + Constants.KAFKA_CLIENTS, CLUSTER_NAME, NAMESPACE).done();
```

Comment:
as we discussed yesterday, you will change this parameters to have only username instead of CLUSTER_NAME, NAMESPACE parameters right ?